test/features/webapp-prompt/webapp-prompt.test.js

```diff
@@ -152,6 +179,26 @@ describe('PEP', () => {
     });
   });
 
+  describe('PEP dismissal tests', () => {
+    it('should add three rings to the app switcher and remove them after the alloted time', async () => {
```

Comment:
super nit: verbs to start test description statements: `adds three rings to the app switcher and removes them after the allotted time`. You can always add `should` to the start of test descriptions because our code always should do something.